applications/dashboard/src/scripts/dom.ts

```diff
@@ -490,3 +491,73 @@ export function watchFocusInDomTree(rootNode: Element, callback: (hasFocus: bool
         }
     });
 }
+
+/**
+ * Sticky header handling
+ */
+function handleStickyHeaderState(element, data) {
+    const goingDown = data.lastScrollPos < data.currentScrollPos;
+    const isAtTopOfPage = data.currentScrollPos === 0;
+    const elementHeight = element.offsetHeight;
+    const isPastHeader =
+        element.style.position !== "fixed" && element.offsetTop + elementHeight <= data.currentScrollPos;
+    const elementTop = element.style.top !== "" ? parseInt(element.style.top, 10) : false;
+
+    element.classList.toggle("isScrollingDown", goingDown);
+    element.classList.toggle("isScrollingUp", !goingDown);
+    element.classList.toggle("isAtTop", isAtTopOfPage);
+
+    if (goingDown) {
+        element.style.position = "";
+        if (isPastHeader) {
+            element.style.top = `${data.currentScrollPos - elementHeight}px`;
+        } else {
+            if (!elementTop) {
+                element.style.top = `${data.currentScrollPos}px`;
+            }
+        }
+    } else {
+        // going UP
+        if (data.currentScrollPos <= elementTop) {
+            element.style.top = "";
+            element.style.position = "fixed";
+        }
+    }
+}
+
+/**
+ * Vanilla's default way to handle sticky headers
+ */
+export function stickyHeader() {
+    const header = document.querySelector(".stickyHeader");
+    if (header !== null) {
+        let currentScrollPos = Math.max(window.scrollY, 0);
+        let lastScrollPos = -1;
+
+        handleStickyHeaderState(header, {
+            currentScrollPos,
+            lastScrollPos,
+        });
+
+        window.addEventListener("scroll", e => {
+            debounce(
+                () => {
+                    window.requestAnimationFrame(data => {
+                        lastScrollPos = currentScrollPos;
+                        currentScrollPos = Math.max(window.scrollY, 0);
+                        handleStickyHeaderState(header, {
+                            currentScrollPos,
+                            lastScrollPos,
+                        });
+                    });
+                },
+                200,
```

Comment:
I was experiencing some jitteryness on my iOS with a 200ms debounce. Setting it 100ms resolves the issue.